src/pages/en/guides/styling.md

```diff
@@ -153,81 +237,19 @@ module.exports = {
 };
 ```
 
-_Note: Astro v0.21 and later requires this manual setup for autoprefixer. Previous versions ran this automatically._
-
-## PostCSS
-
-You can use any [PostCSS plugin](https://www.postcss.parts/) by adding a `postcss.config.cjs` file to the root of your project. Follow the documentation for the plugin you’re trying to install for configuration and setup.
-
----
-
-## Supported Styling Options
-
-Styling in Astro is meant to be as flexible as you'd like it to be! The following options are all supported:
-
-| Framework        | Global CSS | Scoped CSS | CSS Modules |
-| :--------------- | :--------: | :--------: | :---------: |
-| `.astro`         |     ✅     |     ✅     |    N/A¹     |
-| `.jsx` \| `.tsx` |     ✅     |     ❌     |     ✅      |
-| `.vue`           |     ✅     |     ✅     |     ✅      |
-| `.svelte`        |     ✅     |     ✅     |     ❌      |
-
-¹ _`.astro` files have no runtime, therefore Scoped CSS takes the place of CSS Modules (styles are still scoped to components, but don't need dynamic values)_
-
-All styles in Astro are automatically minified and bundled, so you can just write CSS and we'll handle the rest ✨.
-
----
-
-## Frameworks and Libraries
-
-### 📘 React / Preact
-
-`.jsx` files support both global CSS and CSS Modules. To enable the latter, use the `.module.css` extension (or `.module.scss`/`.module.sass` if using Sass).
-
-```js
-import './global.css'; // include global CSS
-import Styles from './styles.module.css'; // Use CSS Modules (must end in `.module.css`, `.module.scss`, or `.module.sass`!)
-```
-
-### 📗 Vue
-
-Vue in Astro supports the same methods as `vue-loader` does:
-
-- [vue-loader - Scoped CSS][vue-scoped]
-- [vue-loader - CSS Modules][vue-css-modules]
-
-### 📕 Svelte
-
-Svelte in Astro also works exactly as expected: [Svelte Styling Docs][svelte-style].
-
-### 🎨 CSS Preprocessors (Sass, Stylus, etc.)
+### Tailwind
```

Comment:
this section can be removed now, and replaced with a reference/link to the new integration README / guides.